docs/index.md

```diff
@@ -50,7 +50,10 @@ This will install uvicorn with "Cython-based" dependencies (where possible) and
 In this context, "Cython-based" means the following:
 
 - the event loop `uvloop` will be installed and used if possible.
+  - `uvloop` is a fast, drop-in replacement of the built-in asyncio event loop. It is implemented in Cython and should always be used in production. Read more [here][uvloop_docs].
```

Comment:
```suggestion
  - `uvloop` is a fast, drop-in replacement of the built-in asyncio event loop. It is implemented in Cython. Read more [here][uvloop_docs].
```

Reply:
Not sure we should be that definitive in the wording about what people should or should not do in production,